teps/0056-pipelines-in-pipelines.md

```diff
@@ -2,283 +2,389 @@
+4. Users should be able to access outputs (`Results`) from the sub-`Pipeline` in the main-`Pipeline`.
+5. Users should be able to access sufficient information about the sub-`PipelineRun` from the status of the
+main-`PipelineRun` to be able to fetch and access the sub-`PipelineRun`'s full status.
+6. Users should be able to propagate actions from the main-`Pipeline` to the sub-`Pipeline`, such as deletion and
+cancellation.
 
 ## References
 
-<!--
-Use this section to add links to GitHub issues, other TEPs, design docs in Tekton
-shared drive, examples, etc. This is useful to refer back to any other related links
-to get more details.
--->
-- [Issue #2134: Support using a PipelineTask in the Pipeline CRD to run other Pipelines the same way we run a Task](https://github.com/tektoncd/pipeline/issues/2134)
-- [Issue #4067: Add a gateway task or grouping for pipelines](https://github.com/tektoncd/pipeline/issues/4067)
-- [Project Proposal](https://github.com/tektoncd/community/issues/330)
-- [Experimental Project](https://github.com/tektoncd/experimental/tree/main/pipelines-in-pipelines)
-- [Original Proposal](https://docs.google.com/document/d/14Uf7XQEnkMFBpNYRZiwo4dwRfW6do--m3yPhXHx4ybk/edit)
+- Issues
+  - [Issue #2134: Support using a PipelineTask in the Pipeline CRD to run other Pipelines the same way we run a Task][issue-2134]
+  - [Issue #4067: Add a gateway task or grouping for pipelines][issue-4067]
+- Experiments
+  - [Experimental Project Proposal][experiment-proposal]
+  - [Pipelines in Pipelines Custom Task][pip]
+- Proposals
+  - [TEP-0044: Data Locality and Pod Overhead in Pipelines][tep-0044]
+  - [TEP-0050: Ignore Task Failures][tep-0050]
+  - [TEP-0059: Skipping Strategies][tep-0059]
+  - [TEP-0084: End-to-End Provenance Collection][tep-0084]
+  - [TEP-0090: Matrix][tep-0090]
+  - [TEP-0100: Embedded TaskRuns and Runs Status in PipelineRuns][tep-0100]
+  - [Original Proposal][doc]
+
+[tep-0044]: 0044-data-locality-and-pod-overhead-in-pipelines.md
+[tep-0050]: 0050-ignore-task-failures.md
+[tep-0059]: 0059-skipping-strategies.md
+[tep-0084]: 0084-endtoend-provenance-collection.md
+[tep-0090]: 0090-matrix.md
+[tep-0100]: 0100-embedded-taskruns-and-runs-status-in-pipelineruns.md
+[pip]: https://github.com/tektoncd/experimental/tree/main/pipelines-in-pipelines
+[doc]: https://docs.google.com/document/d/14Uf7XQEnkMFBpNYRZiwo4dwRfW6do--m3yPhXHx4ybk/edit
+[experiment-proposal]: https://github.com/tektoncd/community/issues/330
+[issue-4067]: https://github.com/tektoncd/pipeline/issues/4067
+[issue-2134]: https://github.com/tektoncd/pipeline/issues/2134
+[when]: https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#guarding-a-task-and-its-dependent-tasks
+[slack]: ../teps/images/0056-slack-thread.png
+[slsa]: https://slsa.dev/
+[scorecard]: https://github.com/ossf/scorecard
+[codeql]: https://github.com/github/codeql
+[matrix-uc]: https://github.com/tektoncd/community/pull/600#pullrequestreview-851817251
```

Comment:
NIT: formatting issue:

This list is not showing at all 🤔 

![Screen Shot 2022-05-06 at 10 20 21 AM](https://user-images.githubusercontent.com/206285/167181581-9ac98e6e-d7a6-41e7-bfcb-3b521ce87d44.png)

Reply:
missing `-`?

Reply:
It's intentional that the lower list doesn't show in markdown - it's a way to provide reusable links in markdown (https://grantwinney.com/cool-markdown-tricks-for-github/#create-reusable-links)

Reply:
great tip, thanks @jerop 👍 

I was looking for the reference to the slack image you have attached and didn't find it in the markdown viewer. Its a great idea to include such screen shots so that the conversation is saved in the repo itself. Without having that reference, its not clearly visible.

This is not a blocker though please merge this!